lib/FFI/Platypus/Lang/Win32.pm

```diff
@@ -39,9 +65,22 @@ sub abi
 
  my $hashref = FFI::Platypus::Lang::Win32->native_type_map;
 
-This returns a hash reference containing the native aliases for the
-Windows API.  That is the keys are native Windows API C types and the
-values are libffi native types.
+This is automatically called when C<< $ffi->lang('Win32'); >> is used.
+It returns definitions for the following native aliases for the Windows API,
+allowing them to be used in function declarations:
+C<BOOL>, C<BOOLEAN>, C<BYTE>, C<CCHAR>, C<CHAR>, C<COLORREF>, C<DWORD>, C<DWORDLONG>,
```

Comment:
I don't want all of these types explicitly listed in the documentation, as it is an extra thing to maintain, and may include dynamic parts in the future.  The old text described exactly what was returned, which I prefer, and if you want to include a note that explains how to get the list of available types you can use this one-liner:

`perl -MFFI::Platypus::Lang::Win32 -E "say for sort keys %{ FFI::Platypus::Lang::Win32->native_type_map }"`